src/components/index.js

```diff
@@ -0,0 +1,5 @@
+export { default as Footer } from "./Footer/Footer";
+export { default as Header } from "./Header/Header";
+export { default as FeaturedBanners } from "./FeaturedBanners/FeaturedBanners";
+export { default as ProductCategories } from "./ProductCategories/ProductCategories";
+export { default as FeaturedProducts } from "./FeaturedProducts/FeaturedProducts";
```

Comment:
Nice component management 👍🏽 . You can avoid the `Component/Component` import if you rename the component file for `index.js` (if possible). This way you can import using `export { default as Footer } from "./Footer"` for example.